The "CLOCK_BOOTTIME_ALARM" and "CLOCK_REALTIME_ALARM"
clocks are not supported by SGX-LKL. SGX-LKL returned error code
"ENOTSUPP(524)".
In the test case, this error code is not checked. Modified the test cases
namely "timer_settime01" and "timer_settime02" to check this return
error code.

diff --git a/testcases/kernel/syscalls/timer_settime/timer_settime01.c b/testcases/kernel/syscalls/timer_settime/timer_settime01.c
index fc1cf86a6..98c2a1892 100644
--- a/testcases/kernel/syscalls/timer_settime/timer_settime01.c
+++ b/testcases/kernel/syscalls/timer_settime/timer_settime01.c
@@ -25,6 +25,10 @@
 #include "tst_test.h"
 #include "lapi/common_timers.h"
 
+#ifndef ENOTSUPP
+#define ENOTSUPP       524
+#endif
+
 static struct timespec timenow;
 static struct itimerspec new_set, old_set;
 static kernel_timer_t timer;
@@ -60,7 +64,7 @@ static void run(unsigned int n)
 
 		TEST(tst_syscall(__NR_timer_create, clock, NULL, &timer));
 		if (TST_RET != 0) {
-			if (possibly_unsupported(clock) && TST_ERR == EINVAL) {
+			if (possibly_unsupported(clock) && (TST_ERR == EINVAL || TST_ERR == ENOTSUPP)) {
 				tst_res(TPASS | TTERRNO,
 					"%s unsupported, failed as expected",
 					get_clock_str(clock));
